internal/hud/resourceview.go

```diff
@@ -75,45 +75,43 @@ type statusDisplay struct {
 	spinner bool
 }
 
-func statusDisplayOptions(res view.Resource) statusDisplay {
-	if res.IsTiltfile {
-		if !res.CurrentBuild.Empty() {
-			return statusDisplay{color: cPending, spinner: true}
-		} else if res.CrashLog.Empty() {
-			return statusDisplay{color: cGood}
-		} else {
-			return statusDisplay{color: cBad}
-		}
-	}
-
-	if !res.CurrentBuild.Empty() && !res.CurrentBuild.Reason.IsCrashOnly() {
+// NOTE: This should be in-sync with combinedStatus in the web UI
```

Comment:
is it worth it/is it easy to put this code in its own package and have both tui and webui import the same function, instead of keeping it in sync in two places?

Reply:
They're implemented in different languages, and have different return values (since the return values are based on what's possible to display in each). so...not easy.

Reply:
Oh whoops I didn't realize the other one was in JS. Okay cool 👍